boms/cloud-oss-bom/pom.xml

```diff
@@ -132,11 +132,6 @@
         <artifactId>grpc-testing</artifactId>
         <version>${io.grpc.version}</version>
       </dependency>
-      <dependency>
-        <groupId>io.grpc</groupId>
-        <artifactId>grpc-okhttp</artifactId>
```

Comment:
Why is this being removed?

Reply:
because it's a dupe

Reply:
Ok. It's confusing to have it bundled in this PR that has a completely unrelated purpose, though. Could you do a separate PR for it? I promise I'll approve it quickly.